Pod/Classes/SLConditionalDateFormatter.m

```diff
@@ -141,9 +141,10 @@ - (BOOL)isInFuture
 @end
 
 
-static inline NSComparisonResult NSCalendarUnitCompareSignificance(NSCalendarUnit a, NSCalendarUnit b)
```

Comment:
The rules for `inline` in C are a little more complicated than in C++, and even in C++ I'd trust the compiler to make the right choice on its own.
I only use `inline` to be able to define functions in header files and include them in several .cpp files without linker errors (which only works in C++), not as an optimisation hint.

> Am 21.08.2015 um 15:20 schrieb <notifications@github.com>:
> 
> In Pod/Classes/SLConditionalDateFormatter.m:
> 
> > @@ -141,9 +141,10 @@ - (BOOL)isInFuture
> >  @end
> >  
> >  
> > -static inline NSComparisonResult NSCalendarUnitCompareSignificance(NSCalendarUnit a, NSCalendarUnit b)
> y u no inline?
> 
> —
> Reply to this email directly or view it on GitHub.
>